eng/templates/stages/release.yml

```
@@ -148,6 +150,18 @@ stages:
             draft='--draft'
           fi
 
+          git tag $releaseTag $dotnetDotnetCommit
+          
+          result=$(git push $publicVmrUrl refs/tags/$releaseTag 2>&1)
+          if [[ $? -ne 0 ]]; then
```

This `-ne` syntax doesn't work with the double brackets.

```suggestion
          if [[ $? != 0 ]]; then
```